### PR TITLE
[dagster-airlift] example revamp

### PR DESCRIPTION
**Pull request by dpeng817**
## Summary & Motivation
- Add a "complete" section with schedule targeting all assets
- "zero in" to a single dag

## How I Tested These Changes
Manual testing + existing unit.

## Changelog
`NOCHANGELOG`

### CODE REVIEW

**Comment by dpeng817** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24583?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24583** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24583?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24582** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24582?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py [hunk 1]
```diff
@@ -7,23 +7,30 @@
...
+        )
     ),
-    lakehouse_existence_check_defs(
-        csv_path=CSV_PATH,
-        duckdb_path=DB_PATH,
+    apply_eager_automation(
+        lakehouse_existence_check_defs(
+            csv_path=CSV_PATH,
+            duckdb_path=DB_PATH,
+        ),
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py:34]: Until we could do this with a one-liner I think it should use a traditional schedule. cc: @OwenKephart for guidance here.

File: examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/oberve_with_check.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+from datetime import timedelta
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/oberve_with_check.py:1]: file name misspelled

**Review by schrockn — CHANGES_REQUESTED** [review]
Mostly about getting to a one-liner with DA.

Why are we changing the dbt project? What happened to jaffle shop? I may have missed something.

**Comment by dpeng817** [comment]
@schrockn 
regarding getting rid of jaffle shop, I felt that once you combined the jobs, jaffle shop started complicating the narrative. IE; why are you materializing all of these unrelated tables in the same job that's loading this particular table? It didn't seem realistic. I also felt that it complicated the scheduling story once you're using declarative automation. My understanding is that you place a cron at the roots, and can have everything default to eager downstream. But for the dbt models that doesn't make as much sense. Since their downstream relationships are different from model to model. There's ways around it, but I felt that all of this was obfuscating from the original goal, which is, show a task expanded out into multiple assets.

**Comment by dpeng817** [comment]
Decided to stick with jaffle shop for the time being. If we're just targeting every asset with a schedule it's not as important.

File: examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py (hunk 1 above)
Comment (OwenKephart) [reply examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py]: Just put up: https://github.com/dagster-io/dagster/pull/24613, which enables setting a default condition on an AutomationConditionSensorDefinition: https://github.com/dagster-io/dagster/pull/24613/files#diff-e24091c0e2d196b1261e2510fb16c2ce7153fad761fb91d843403cc5c0a3e2c7R18

File: examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/oberve_with_check.py (hunk 2 above)
Comment (dpeng817) [reply examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/oberve_with_check.py]: whoops. fixed.

File: examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+from dbt_example.dagster_defs.constants import dbt_manifest_path, dbt_project_path
 from dbt_example.dagster_defs.lakehouse import (
     defs_from_lakehouse,
     lakehouse_existence_check_defs,
     specs_from_lakehouse,
 )
 from dbt_example.shared.load_iris import CSV_PATH, DB_PATH, IRIS_COLUMNS
 
-from .constants import dbt_manifest_path, dbt_project_path
+daily_schedule = ScheduleDefinition(name="daily_schedule", cron_schedule="0 0 * * *", target="*")
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py:13]: 👍🏻 much simpler for now

**Review by schrockn — APPROVED** [review]
cool. More comfortable with this more conservative approach for now and we can iterate.

**Pull request merged**